fbw-a380x/src/systems/instruments/src/MFD/pages/SURV/MfdSurvControls.tsx

```diff
@@ -250,21 +253,23 @@ export class MfdSurvControls extends DisplayComponent<MfdSurvControlsProps> {
               <div class="mfd-surv-controls-tcas-left">
                 <RadioButtonGroup
                   values={['TA/RA', 'TA ONLY', 'STBY']}
+                  onModified={(val) =>
+                    this.props.bus.getPublisher<MfdSurvEvents>().pub('tcas_alert_level', 2 - val, true)
+                  }
                   selectedIndex={this.tcasTaraSelectedIndex}
                   idPrefix={`${this.props.mfd.uiService.captOrFo}_MFD_survControlsTcasTara`}
                   additionalVerticalSpacing={10}
                   color={Subject.create('green')}
-                  valuesDisabled={Subject.create(Array(3).fill(true))}
                 />
               </div>
               <div class="mfd-surv-controls-tcas-right">
                 <RadioButtonGroup
                   values={['NORM', 'ABV', 'BLW']}
                   selectedIndex={this.tcasNormAbvBlwSelectedIndex}
+                  onModified={(val) => this.props.bus.getPublisher<MfdSurvEvents>().pub('tcas_alt_select', val, true)}
```

Comment:
As above, but sub to this.tcasNormAbvBlwSelectedIndex.

Reply:
Should be in the TCAS PR, also there is a bug there that I wasn't able to solve with this change, so it's not done yet.